site/src/xServices/template/searchUsersAndGroupsXService.ts

```diff
@@ -1,101 +0,0 @@
-import { getGroups, getTemplateACLAvailable, getUsers } from "api/api";
-import { Group, User } from "api/typesGenerated";
-import { queryToFilter } from "utils/filters";
-import { everyOneGroup } from "utils/groups";
```

Comment:
It looks like these utils are unused now so we could delete them.

Reply:
They are tho, the queryToFilter is being used in other places and I should use the everyOneGroup... good catch!

Reply:
Oh weird, I tested and the everyone group was showing up for me even without it.

Reply:
You are totally right!